petrinet/examples/halfar.json

```diff
@@ -0,0 +1,258 @@
+    "model_version": "0.1"
+  },
+  "model": {
+    "states": [
+      {
+        "id": "h_0",
+        "name": "h_0",
+        "description": "height"
+      },
+      {
+        "id": "h_1",
+        "name": "h_1",
+        "description": "height"
+      },
+      {
+        "id": "h_2",
+        "name": "h_2",
+        "description": "height"
+      },
+      {
+        "id": "h_3",
+        "name": "h_3",
+        "description": "height"
+      },
+      {
+        "id": "h_4",
+        "name": "h_4",
+        "description": "height"
+      }
+    ],
+    "transitions": [
+      {
+        "id": "w_n_0",
+        "input": [
+          "h_1",
+          "h_0"
+        ],
+        "output": [
+          "h_0"
+        ],
+        "properties": {
+          "name": "w_n_0"
+        }
+      },
+      {
+        "id": "w_p_0",
+        "input": [
+          "h_2",
+          "h_1"
+        ],
+        "output": [
+          "h_0"
+        ],
+        "properties": {
+          "name": "w_p_0"
+        }
+      },
+      {
+        "id": "w_n_1",
+        "input": [
+          "h_2",
+          "h_1"
+        ],
+        "output": [
+          "h_1"
+        ],
+        "properties": {
+          "name": "w_n_1"
+        }
+      },
+      {
+        "id": "w_p_1",
+        "input": [
+          "h_3",
+          "h_2"
+        ],
+        "output": [
+          "h_1"
+        ],
+        "properties": {
+          "name": "w_p_1"
+        }
+      },
+      {
+        "id": "w_n_2",
+        "input": [
+          "h_3",
+          "h_2"
+        ],
+        "output": [
+          "h_2"
+        ],
+        "properties": {
+          "name": "w_n_2"
+        }
+      },
+      {
+        "id": "w_p_2",
+        "input": [
+          "h_4",
+          "h_3"
+        ],
+        "output": [
+          "h_2"
+        ],
+        "properties": {
+          "name": "w_p_2"
+        }
+      },
+      {
+        "id": "w_n_3",
+        "input": [
+          "h_2",
+          "h_3"
+        ],
+        "output": [
+          "h_3"
+        ],
+        "properties": {
+          "name": "w_n_3"
+        }
+      },
+      {
+        "id": "w_p_3",
+        "input": [
+          "h_4",
+          "h_3"
+        ],
+        "output": [
+          "h_3"
+        ],
+        "properties": {
+          "name": "w_p_3"
+        }
+      },
+      {
+        "id": "w_n_4",
+        "input": [
+          "h_2",
+          "h_3"
+        ],
+        "output": [
+          "h_4"
+        ],
+        "properties": {
+          "name": "w_n_4"
+        }
+      },
+      {
+        "id": "w_p_4",
+        "input": [
+          "h_3",
+          "h_4"
+        ],
+        "output": [
+          "h_4"
+        ],
+        "properties": {
+          "name": "w_p_4"
+        }
+      }
+    ]
+  },
+  "semantics": {
+    "ode": {
+      "rates": [
+        {
+          "target": "w_n_0",
+          "expression": "-1*gamma*(h_1-h_0)^3*h_0^5"
```

Comment:
One more small issue I noticed is that downstream (at least our processing pipeline) `**` is expected for exponentiation, not `^`. Could that be changed?

Reply:
Yes, no problem.